api/interservice/authz/v2/project.proto

```diff
@@ -94,6 +94,7 @@ message ProjectRules {
     repeated ProjectRule rules = 2;
 }
 message ProjectRule {
+    string type = 1;
```

Comment:
Each project has a collection of rules. Each rule has a type of 'node' or 'event' and a collection of conditions.

Reply:
![tenor-114117122](https://user-images.githubusercontent.com/1679247/56386552-f0103380-61d6-11e9-8c88-488b2e348059.gif)
I love the idea of using enum here. I am also going to try to update the condition's type to be an enum also.